CONTRIBUTING.md

```diff
@@ -1,96 +1,60 @@
-- [ ] The pull request does not contain uses of `any` or `{}` without comments describing why.
-- [ ] The pull request does not contain large React components that could easily be split into several smaller components.
-- [ ] The pull request does not contain back end calls directly from components, use actions and Redux instead.
-- [ ] The pull request follows our [styling with Emotion convention](./style_guides/styling.md)
-  > We still use a lot of SASS, but any new CSS work should be using or migrating existing code to Emotion
+## Where do I go from here?
 
-#### Redux specific checks (skip if your pull request does not contain Redux changes)
-
-- [ ] The pull request does not contain code that mutates state in reducers or thunks.
-- [ ] The pull request uses helpers `actionCreatorFactory` and `reducerFactory` instead of traditional `switch statement` reducers in Redux. See [Redux framework](https://github.com/grafana/grafana/tree/master/style_guides/redux.md) for more details.
-- [ ] The pull request uses `reducerTester` to test reducers. See [Redux framework](https://github.com/grafana/grafana/tree/master/style_guides/redux.md) for more details.
-- [ ] The pull request does not contain code that accesses the reducers state slice directly, instead, the code uses state selectors to access state.
-
-### Pull request titles and message
-
-Pull request titles should follow this format: `Area: Name of the change`. Titles are used to generate the changelog so they should be as descriptive as possible in one line.
-
-Good examples:
-
-- `Explore: Adds Live option for supported data sources`
-- `GraphPanel: Don't sort series when legend table & sort column is not visible`
-- `Build: Support publishing MSI to grafana.com`
-
-The message in the pull requests should contain a reference so the issue if there is one. For example, `Closes #<issue number>`, `Fixes #<issue number>`, or `Ref #<issue number>` if the change is related to an issue but does not close it. Make sure to explain what problem the pull request is solving and why its implemented this way. As a new contributor its often better to overcommunicate to avoid back-and-forth communication, as it consumes time and energy.
-
-### Git commit formatting
-
-Grafana Squash Pull requests when merging them into master. This means the maintainer will be responsible for the title in the git commit message.
-The commit message of the commits in the Pull Request can still be part of the git commit body. So it's always encouraged to write informative commit messages.
-
-The Git commit title should be short, descriptive and include the Pull Request ID.
-
-Good examples:
-
-- `Explore: Live supprt in data sources (#12345)`
-- `GraphPanel: Fix legend sorting issues (#12345)`
-- `Build: Support publishing MSI to grafana.com (#12345)`
-
-Its also good practice to include a reference to the issue in the Git commit body when possible.
+- Set up your [development environment](contribute/development.md).
```

Comment:
```suggestion
- Set up your [development environment](contribute/development.md)
```

Reply:
I think it's actually the other items should end with a period.

Reply:
Giving this a second read. I agree. Let me fix that!